src/main/java/com/connexta/store/exceptions/IngestException.java

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright (c) 2019 Connexta, LLC
+ *
+ * Released under the GNU Lesser General Public License version 3; see
+ * https://www.gnu.org/licenses/lgpl-3.0.html
+ */
+package com.connexta.store.exceptions;
+
+import com.connexta.store.exceptions.common.DetailedResponseStatusException;
+import org.springframework.http.HttpStatus;
+
+public class IngestException extends DetailedResponseStatusException {
```

Comment:
Is this ever used? Can we get away with just a TranformException?

Reply:
Only for transform exception to extend 🤔 I will remove